packages/svelte/tests/migrate/samples/slot-non-identifier/output.svelte

```diff
@@ -44,4 +44,35 @@
 			cool
 		
 	{/snippet}
+</Comp>
+
+
+<!-- don't remove the let directive if we don't migrate -->
+
+<Comp>
+	<!-- @migration-task: migrate this slot by hand, `cool:stuff` is an invalid identifier -->
+	<div let:shoudl_stay slot="cool:stuff">
```

Comment:
are all the "shoudl" in this file meant to be "shoudl"?

Reply:
Obviously lol...i can type 😄